.changeset/sweet-queens-roll.md

```diff
@@ -0,0 +1,26 @@
+---
+'@graphql-yoga/e2e-testing': patch
+'aws-lambda': patch
+'aws-lambda-bundle': patch
+'azure-function': patch
+'cloudflare-advanced': patch
+'cloudflare': patch
+'file-upload-nextjs-pothos': patch
+'file-upload-nexus': patch
+'example-generic-auth': patch
+'example-graphql-armor': patch
+'graphql-ws': patch
+'hackernews': patch
+'issue-template': patch
+'example-live-query': patch
+'nextjs': patch
+'nextjs-auth': patch
+'example-redis-pub-sub': patch
+'service-worker': patch
+'sveltekit': patch
+'vercel-function-bundle': patch
+'@graphql-yoga/common': patch
+'@graphql-yoga/node': patch
```

Comment:
Examples and actual packages must be in separate changesets (or as @dotansimha suggested just remove examples from the bumps)